src/tests/functional/shared_test_classes/include/shared_test_classes/single_op/nms_rotated.hpp

```diff
@@ -35,7 +35,7 @@ class NmsRotatedOpTest : public testing::WithParamInterface<NmsRotatedParams>,
 
 protected:
     void SetUp() override;
-
+    void compare(const std::vector<ov::Tensor> &expected, const std::vector<ov::Tensor> &actual) override;
```

Comment:
Is it possible not overload the compare method and place comparation for these4 op in base class -> comparation_map?